lib/init.dart

```diff
@@ -28,4 +29,12 @@ void init() {
   GetIt.instance.registerSingleton(CategoryApi());
   GetIt.instance.registerSingleton(ProductApi());
   GetIt.instance.registerSingleton(ProductListApi());
+
+  GetIt.instance.registerSingletonAsync(() async {
```

Comment:
As mentioned previously, do we want to register this prior to our Api's (which rely on the path configs - in theory)?

Reply:
Hmm right, I put it after without thinking a lot about it, about API < Repository < Bloc

But if the API relies on it, there is a dilemma 🤔